content/02.main-text.md

```diff
@@ -222,22 +218,33 @@ manubot cite --render --format=markdown \
 
 ## Authorship
 
+Manubot does not impose any restrictions on authorship.
+It allows authors to adhere to the author inclusion and author ordering conventions of their field, which vary considerably across disciplines [@doi:10.1371/journal.pone.0023477].
+Some Manubot projects create a [table](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) in their GitHub repository to track contributors who did not commit text to the manuscript.
+This provides a transparent way to record contributions such as experimental research that generated data for the manuscript and discuss whether they meet that project's authorship criteria.
+Contribution transparency helps prevent ghostwriting [@doi:10.1371/journal.pmed.1000023] and is especially important in collaborative writing [@doi:10.1371/journal.pcbi.1006508].
+Although we recommend authors provide their ORCID and GitHub username, Manubot also supports pseudonyms, pseudonymous GitHub usernames, and authors without an ORCID or GitHub account.
+
 To determine authorship for the Deep Review, we followed the International Committee of Medical Journal Editors (ICMJE) [guidelines](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html) and used GitHub to track contributions.
 ICMJE recommends authors substantially contribute to, draft, approve, and agree to be accountable for the manuscript.
 We acknowledged other contributors who did not meet all four criteria, including contributors who provided text but did not review and approve the complete manuscript.
 Although these criteria provided a straightforward, equitable way to determine who would be an author, they did not produce a traditionally ordered author list.
 In biomedical journals, the convention is that the first and last authors made the most substantial contributions to the manuscript.
 This convention can be difficult to reconcile in a collaborative effort.
 Using git, we could quantify the number of commits each author made or the number of sentences an author wrote or edited, but these metrics discount intellectual contributions such as discussing primary literature and reviewing pull requests.
-However, there is no objective system to compare and weight the different types of contributions and produce an ordered author list.
+Therefore, we concluded that it is not possible to construct an objective system to compare and weight the different types of contributions and produce an ordered author list.
```

Comment:
My personal opinion is that it is possible to design a system for ranking contributors designed to be objective. However, it would be a very large task with potential downsides, like perhaps being a bit petty to potentially spend hours of human time resolving small author order differences. Is there a way this sentence could capture that there was no readily actionable system... or no system that was worth the hassle.

Reply:
My recollection from deep review was that there wasn't consensus about how we could convert different types of contributions.  There are ways to quantify contributions and convert them into rankings (I saw one in an article from the 90s yesterday but lost the link), but I consider them to be subjective because different individuals may develop different schemes.  I also expect that any fixed quantitative system would be gameable, which is why I went for a strong assertion that no objective system exists.

Do you still think we should back off that claim?

Reply:
> but I consider them to be subjective because different individuals may develop different schemes

I agree. Picture worth a thousand words, and all that...

Reply:
The system that comes to mind is to have all co-authors rank all contributions and then have an algorithm produce a consensus ranking. Seems objective to me if you trust contributors to rank others fairly. Obviously there could be vote selling.

I am fine with existing phrasing, especially since that captures why the decision was made at that time.

Reply:
@dhimmel if you're okay with this, I'll merge.  Feel free to edit again in a new PR if you'd like.